test_scripts/Defects/7_0/1910_Navigation_app_must_get_LIMITED_and_AUDIBLE_in_case_embedded_audio_source_is_activated_and_MixingAudioSupported_true.lua

```diff
@@ -0,0 +1,62 @@
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1910
```

Comment:
should be added to 7_0 defects test_set?

Reply:
Good catch, fixed